### ISSUE TITLE
Temporarily disable notifications for a channel with an active discussion for X minutes or hours

### ISSUE THREAD

**Issue by raphj** [matrix-org/matrix-spec#1075]
(I've initially reported this for Element at https://github.com/vector-im/element-web/issues/22289 (marked as 0-Uncommon). Submitting this here as [suggested](https://github.com/vector-im/element-web/issues/22289#issuecomment-1132657137) by @t3chguy. I'm not familiar enough with the Matrix protocol to go into technical details.)

#### What would you like to do?

I would like to be able to temporary mute a channel for a few minutes. 

#### Why would you like to do it?

I usually want to see all the notification messages from all the channels I'm in. The traffic is quite low overall. However, it happens that some channel has an ongoing active discussion that I'm not particularly interested in and that produces a lot of notifications. It'd be nice to temporarily ignore this channel until the ongoing discussion settles down to avoid missing focus on my current task or missing more important notifications.

#### How would you like to achieve it?

Clicking on a menu item "Disable notification for some time", with sub items like "10 minutes", "30 minutes", "1 hour", "2 hours" in the contextual menu of a channel.

### Have you considered any alternatives?

I considered manually disabling notifications for the channel and thinking of re-enabling them when later, but I risk forgetting to re-enable them.

**Comment by t3chguy** [comment]
Related https://github.com/matrix-org/matrix-spec-proposals/pull/3767

**Comment by gabrc52** [comment]
Agreed, this is how Signal does it:

![image](https://github.com/matrix-org/matrix-spec/assets/24363938/5b3e840a-2dc0-47ec-8e36-01077d377705)

> I considered manually disabling notifications for the channel and thinking of re-enabling them when later, but I risk forgetting to re-enable them.

And this is bad for the Matrix ecosystem because if you forget to re-enable notifications you will fail to respond to messages in those rooms. This is why this and #612 are crucial features to have, since they overall increase user message responsiveness.

**Comment by unmotivatedgene** [comment]
I also wish to temporarily mute a conversation. I keep forgetting to unmute and leave them unread for days...

**Comment by HammyHavoc** [comment]
> I also wish to temporarily mute a conversation. I keep forgetting to unmute and leave them unread for days... 

Yes, I personally experience this exact same pattern, and am on the receiving end of it with others.

It's quite frustrating, because between this and a WhatsApp-like `archived` status for conversations, Matrix really *is* largely there for the most part in 2026. Been loving it for about half a decade. Whenever people ask me what it lacks versus the normal comms solutions, it's those two things every time for me.
